src/screens/ConstructionSiteOverviewScreen.js

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
+import React, { useCallback, useContext } from 'react';
+import { FlatList, StyleSheet, TouchableOpacity, View } from 'react-native';
+import { ListItem } from 'react-native-elements';
+
+import { BoldText, Icon, RegularText, SafeAreaViewFlex } from '../components';
+import { colors, consts, normalize } from '../config';
+import { ConstructionSiteContext } from '../ConstructionSiteProvider';
+import { momentFormat } from '../helpers';
+import { useMatomoTrackScreenView } from '../hooks';
+import { arrowLeft, arrowRight } from '../icons';
+
+const { MATOMO_TRACKING } = consts;
+
+export const ConstructionSiteOverviewScreen = ({ navigation }) => {
+  const { constructionSites } = useContext(ConstructionSiteContext);
+
+  useMatomoTrackScreenView(MATOMO_TRACKING.SCREEN_VIEW.CONSTRUCTION_SITE_OVERVIEW);
+
+  const renderItem = useCallback(
+    ({ index, item }) => {
+      const onPress = () => navigation.navigate('ConstructionSiteDetail', { index });
+
+      const { category, endDate, startDate, title } = item;
+
+      const extendedTitle = (category ? `${category} - ` : '') + title;
+      const formattedStartDate = momentFormat(startDate);
+
+      const formattedEndDate = endDate ? momentFormat(endDate) : undefined;
+      const formattedDates =
+        !formattedEndDate || formattedEndDate === formattedStartDate
+          ? formattedStartDate
+          : `${formattedStartDate} - ${formattedEndDate}`;
+
+      return (
+        <ListItem
+          onPress={onPress}
+          rightIcon={<Icon xml={arrowRight(colors.darkText)} />}
+          subtitle={<RegularText>{formattedDates}</RegularText>}
+          title={<BoldText>{extendedTitle}</BoldText>}
```

Comment:
to match existing list items, the first line of the list item should be a small normal text with the date and the second line the bold title.
currently it is switched.

|construction|event|
|-------------|-----|
|![Bildschirmfoto 2021-01-08 um 12 42 33](https://user-images.githubusercontent.com/1942953/104011875-1ba72e80-51af-11eb-81a1-1d5d994589c9.png)|![Bildschirmfoto 2021-01-08 um 12 42 24](https://user-images.githubusercontent.com/1942953/104011882-1f3ab580-51af-11eb-9de2-b0478bfe8875.png)|

Reply:
done in f77e0d4bd76de4c86fe68e8d94782584b22209f8

Reply:
new screenshot would be nice to not have to run it again on my side